index.toml

```diff
@@ -4,29 +4,31 @@ toc = true
 colab = "https://colab.research.google.com/github/deepset-ai/haystack-tutorials/blob/main/tutorials/"
 
 [[tutorial]]
-title = "Build Your First QA System"
+title = "Build Your First Question Answering System"
 description = "Get Started by creating a Retriever Reader pipeline."
 level = "beginner"
 weight = 10
-notebook = "01_Basic_QA_Pipeline.ipynb"
+notebook = "01_build_your_first_question_answering_system.ipynb"
 aliases = ["first-qa-system"]
-slug = "01_Basic_QA_Pipeline"
+slug = "01_basic_qa_pipeline"
 
 [[tutorial]]
-title = "Fine-Tuning a Model on Your Own Data"
+title = "Fine-Tune a Reader"
 description = "Improve the performance of your Reader by performing fine-tuning."
 level = "intermediate"
 weight = 50
-notebook = "02_Finetune_a_model_on_your_data.ipynb"
-aliases = ["fine-tuning-a-model"]
+notebook = "02_finetune_a_reader.ipynb"
+aliases = ["fine-tuning-a-model", "02_finetune_a_model_on_your_data", "fine-tune-a-model"]
+slug = "02_finetune_a_model_on_your_data"
 
 [[tutorial]]
-title = "Build a QA System Without Elasticsearch"
-description = "Create a Retriever Reader pipeline that requires no external database dependencies."
+title = "Build a Scalable Question Answering System"
+description = "Create a scalable Retriever Reader pipeline that uses an ElasticsearchDocumentStore."
 level = "beginner"
 weight = 15
-notebook = "03_Basic_QA_Pipeline_without_Elasticsearch.ipynb"
+notebook = "03_build_a_scalable_question_answering_system.ipynb"
 aliases = ["without-elasticsearch"]
+slug = "03_scalable_qa_pipeline"
```

Comment:
So now that we no longer have and no longer want to maintain "03_build_a_scalable_question_answering_system" --> where do you want the people looking for this to go to? do you want them to be directed to the 1st tutorial? Or this tutorial? We can add the old slug to one of the aliases accordingly.

Reply:
the first tutorial, I'd say